src/main/webapp/js/omniComment.js

```diff
@@ -256,21 +255,19 @@ $(document).ready(function() {
     }
     
     function hideCommentAndItsPanel(comment) {
-        $(comment).hide();
+        var commentToHide = $(comment);
+        commentToHide.hide();
         
         // hide comment's add form in commentListRegionForFeedbackResponse
         $("li[id^='showResponseCommentAddForm']").hide();
         
         // to hide feedback question + feedback session panel
         // if all list elements are hidden within fbResponse, then hide fbResponse
-        if ($(comment).prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
-            if ($(comment).parent().find('li[style*="display: none"]').length === $(comment).parent().find('li').length) {
-                var commentListRegionForFeedbackResponse = $(comment).closest('tr');
-                // a fbResponse in instructorCommentsPage (html) is made up of 4 rows as the followings
-                commentListRegionForFeedbackResponse.hide();
-                commentListRegionForFeedbackResponse.prev().hide();
-                commentListRegionForFeedbackResponse.prev().prev().hide();
-                commentListRegionForFeedbackResponse.prev().prev().prev().hide();
+        if (commentToHide.prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
+            if (commentToHide.parent().find('li[style*="display: none"]').length === commentToHide.parent().find('li').length) {
```

Comment:
These two lines look similar to the previous case. If they are, may be extract as function?